### PR TITLE
[BGP&IPv6]Add ping in test_bgp_speaker_announce_routes_v6

### PR DESCRIPTION
**Pull request by bingwang-ms**
### Description of PR

Summary:
Fixes # (issue)
This PR fix test failure of test_bgp_speaker_announce_routes_v6 by adding ping nexthop_v6. 
When there is no arp entry for next hop, routes learnt from exabgp will not be set down to ASIC. Therefore, this commit add a ping6 command for each next hop in ipv6 before ipv6 routes are announced.
### Type of change

- [x] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

#### What is the motivation for this PR?
This PR is to fix the failure of test_bgp_speaker_announce_routes_v6. When there is no arp entry for next hop, routes learnt from exabgp will not be set down to ASIC. This PR work around this issue with ping6 command to each next hop in ipv6.
#### How did you do it?
Add a ping6 command to each next hop in ipv6 before ipv6 routes are announced.
#### How did you verify/test it?
Verified on Arista-7260 after clear arp cache and ipv6 neighbor cache.
```
platform linux2 -- Python 2.7.12, pytest-4.6.5, py-1.8.1, pluggy-0.13.1 -- /usr/bin/python
cachedir: .pytest_cache
ansible: 2.8.7
rootdir: /data/Networking-acs-sonic-mgmt/tests, inifile: pytest.ini
plugins: forked-1.1.3, xdist-1.28.0, ansible-2.2.2, repeat-0.8.0
collected 3 items                                                                                                                                                                                     

bgp/test_bgp_speaker.py::test_bgp_speaker_bgp_sessions PASSED                                                                                                                                   [ 33%]
bgp/test_bgp_speaker.py::test_bgp_speaker_announce_routes[True-False-1514] PASSED                                                                                                               [ 66%]
bgp/test_bgp_speaker.py::test_bgp_speaker_announce_routes_v6[False-True-1514] PASSED                                                                                                            [100%]

------------------------------------------------------------------ generated xml file: /data/Networking-acs-sonic-mgmt/tests/tr.xml -------------------------------------------------------------------
===================================================================================== 3 passed in 178.33 seconds ======================================================================================
```
#### Any platform specific information?
NO
#### Supported testbed topology if it's a new test case?
N/A
### Documentation 

N/A

### CODE REVIEW

**Comment by bingwang-ms** [comment]
> @bingwang-ms I have a question here.
> What sonic image did you use?
> I tested my PR with 201811.83 and 201911.(latest two weeks ago) and that worked on 6100 platform.
> Also vs tested it in PR and the test worked smoothly.
> Now we're seeing a bug in the test.
> #1944

I tested on an Arista-7260 DUT, and the SONiC image version is SONiC Software Version: SONiC.20191130.45.
The test_bgp_speaker_announce_routes_v6 will fail if there is no neighbor info for fc02:1000::2/3/4. But it will pass after I run 
ping -6 fc02:1000::X. I noticed the commend Line 122-125, and guess that it may cause by similar reason.
```
        duthost.command("ip route flush %s/32" % ip.ip)
        # The ping here is workaround for known issue:
        # https://github.com/Azure/SONiC/issues/387 Pre-ARP support for static route config
        # When there is no arp entry for next hop, routes learnt from exabgp will not be set down to ASIC
        # Also because of issue https://github.com/Azure/sonic-buildimage/issues/5185 ping is done before route addition.
        duthost.shell("ping %s -c 3" % ip.ip)

```

File: tests/bgp/test_bgp_speaker.py [hunk 1]
```diff
@@ -95,11 +95,14 @@ def common_setup_teardown(duthost, ptfhost, localhost):
     vlan_if_name = vlan_ipv6_entry['attachto']
     nexthops_ipv6 = generate_ips(3, vlan_ipv6_prefix, [IPAddress(vlan_ipv6_address)])
     logging.info("Generated nexthops_ipv6: %s" % str(nexthops_ipv6))
-
+    logging.info("setup ip/routes in ptf")
+    for i in [0, 1, 2]:
+        ptfhost.shell("ip -6 addr add %s dev eth%d:%d" % (nexthops_ipv6[i], vlan_ports[0], i))
     # Set ipv6 nexthop addresses on the ptf interfaces
+    duthost.command("ip -6 route flush %s" % vlan_ipv6_prefix)
+    duthost.command("ip -6 route add %s dev %s" % (vlan_ipv6_prefix, vlan_if_name))
```
[the remark this answers is not in the document]
Comment (bingwang-ms) [reply tests/bgp/test_bgp_speaker.py:103]: Not really. There is no route for IPv6 on the DUT.
```
admin@str-7260cx3-acs-1:~$ ip -6 route show match fc02:1000::
default via fc00::22 dev PortChannel0001 proto 186 src fc00:1::32 metric 20  pref medium
default via fc00::26 dev PortChannel0002 proto 186 src fc00:1::32 metric 20  pref medium
default via fc00::2a dev PortChannel0003 proto 186 src fc00:1::32 metric 20  pref medium
default via fc00::2e dev PortChannel0004 proto 186 src fc00:1::32 metric 20  pref medium
``` 
In order to run the ping6 command, I added static route here.

**Comment by lguohan** [comment]
@pavel-shirshov, is this because you did not add the vlan ipv6 address for this t0-112 topology?

**Comment by bingwang-ms** [comment]
@pavel-shirshov Hi Pavel, could you help to confirm?

**Comment by pavel-shirshov** [comment]
@lguohan I added prefixes to 116
https://github.com/Azure/sonic-mgmt/pull/1982/files#diff-2311d16f9455ea9d08e6a22752de53eeR144

**Comment by keboliu** [comment]
we also see this test failure with the recent 201911 community images.

**Comment by pavel-shirshov** [comment]
@keboliu Can you please check that your DUT has ipv6 addresses assigned to vlan interfaces?

File: tests/bgp/test_bgp_speaker.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (bingwang-ms) [reply tests/bgp/test_bgp_speaker.py:103]: Yes. I believe so.

**Pull request merged**